org.doom2d.forever.yml

```diff
@@ -0,0 +1,96 @@
+  - --share=network
+modules:
+  - name: gme
+    buildsystem: cmake-ninja
+    sources:
+      - type: archive
+        url: https://bitbucket.org/mpyne/game-music-emu/downloads/game-music-emu-0.6.3.tar.gz
+        sha256: 626e8a104e0dadd10ef6519a67aca880c7b40f81471659f1935b61754e12fc7b
+        x-checker-data:
+          type: anitya
+          project-id: 866
+          url-template: https://bitbucket.org/mpyne/game-music-emu/downloads/game-music-emu-$version.tar.gz
+  - name: libmodplug
+    sources:
+      - type: archive
+        url: https://downloads.sourceforge.net/modplug-xmms/libmodplug-0.8.9.0.tar.gz
+        sha256: 457ca5a6c179656d66c01505c0d95fafaead4329b9dbaa0f997d00a3508ad9de
+        x-checker-data:
+            type: anitya
+            project-id: 5669
+            url-template: https://downloads.sourceforge.net/modplug-xmms/libmodplug-$version.tar.gz
+  - name: libxmp
+    sources:
+      - type: archive
+        url: https://github.com/libxmp/libxmp/releases/download/libxmp-4.5.0/libxmp-4.5.0.tar.gz
+        sha256: 7847d262112d14e8442f44e5ac6ed9ddbca54c251284720b563c852b31f26e75
+  - name: enet
+    sources:
+    - type: git
+      url: https://github.com/lsalzman/enet.git
+      branch: master
+    - type: script
+      dest-filename: autogen.sh
+      commands:
+        - autoreconf -ifv
+  - name: d2df-sdl
+    sources:
+    - type: git
+      url: https://github.com/Challenge9/doom2d-forever.git
+      branch: master
+    - type: file
+      url: https://github.com/Challenge9/DF-Res/releases/download/v0.1.0/doomer.wad
+      sha256: "d158d0604d70fe2f9acdd8e001a8d7f3db194ebaea0c6e85e5f4f4b9a3c8e727"
+    - type: file
+      url: https://github.com/Challenge9/DF-Res/releases/download/v0.1.0/game.wad
+      sha256: "63966ec4d2bf77838308dafe03f404535cd5ce2efb81ac222946d91dd414f002"
+    - type: file
+      url: https://github.com/Challenge9/DF-Res/releases/download/v0.1.0/standart.wad
+      sha256: "62755faa8c05718aca55f6dc4f14d9c5342763b37102bb66093276a7240b0d91"
+    - type: file
+      url: https://github.com/Challenge9/DF-Res/releases/download/v0.1.0/shrshade.wad
+      sha256: "e1039b7d991c1fb5a078604fde9ecedb5e6acbaee72c3d8646f1677a16d9045d"
+    - type: file
+      url: https://github.com/Challenge9/DF-Res/releases/download/v0.1.0/doom2d.wad
+      sha256: "be9d691439ab5a9ef842b64b89de17fbe6f08f85cdc916f7294543410d6c64ba"
+    - type: file
+      path: doom2df.desktop
+    - type: file
+      path: doom2df.png
+    buildsystem: simple
+    build-commands:
+      - (export FLATPAK_APP_NAME="$FLATPAK_ID"; export D2DF_BUILD_HASH="$(git rev-parse HEAD)"; cd src/game; fpc -B -dUSE_OPENAL -dUSE_VORBIS -dUSE_MPG123 -dUSE_OPUS -dUSE_MODPLUG -dUSE_XMP -dUSE_GME -dUSE_MINIUPNC Doom2DF.lpr)
+      - install -D src/game/Doom2DF $FLATPAK_DEST/bin
+      - mkdir -p /app/share/doom2df/{wads,data,data/models,maps,maps/megawads}
+      - cp doom2d.wad /app/share/doom2df/maps
+      - mv doom2d.wad /app/share/doom2df/maps/megawads
```

Comment:
why do you duplicated this file?

Reply:
In the game, `megawads` stands roughly for single player campaigns, and files in `maps` directory are all other maps. As this distribution has only 1 map (that is, the standard campaign), I thought it would be reasonable to also duplicate it in `maps`. Thinking about it now, it really is unnecessary.